lib/screens/tree_list/tree_list_view_model.dart

```diff
@@ -21,24 +24,29 @@ abstract class TreeListViewModelBase with Store, ViewModel {
       DependecyInjection.instance.get<TreeStoreBase>();
 
   final GetTreeList _useCase = DependecyInjection.instance.get<GetTreeList>();
+  final ConnectionStatus connectionStatus =
+      DependecyInjection.instance.get<ConnectionStatus>();
 
   @observable
   bool isLoadingTrees = false;
+  bool hasInternetConnection = false;
+  List<Tree> newTrees = [];
 
   @override
   void init() {
-    if (treeStore.isTreeListEmpty()) {
-      fetch();
-    }
+    activateConnectionManager();
   }
```

Comment:
Delete the activation in here (Already made in the "main_common" file). 
And why delete the initial fetch   ?